src/main/java/com/hm/achievement/listener/PlayerAdvancedAchievementListener.java

```diff
@@ -1,10 +1,8 @@
 package com.hm.achievement.listener;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
+import java.util.*;
```

Comment:
Wildcard imports are often frowned upon and they're making the changelog unnecessarily chatty here.